tests/MatrixTest.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\math\tests;
+
+
+use pocketmine\math\Matrix;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class for the Matrix class unit test
+ */
+class MatrixTest extends TestCase{
+
+	public function testConstructor(){
+		$matrix = new Matrix(1, 2);
+
+		$this->assertSame(1, $matrix->getRows());
```

Comment:
not sure why assert that the row count is correct but not the column count? seems like a logic hole.

Reply:
You're right, I add the ```getColumn``` to assert the matrix column :-).